web/templates/invite_user_modal.hbs

```diff
@@ -2,6 +2,9 @@
     {{#if development_environment}}
     <div class="alert" id="dev_env_msg"></div>
     {{/if}}
+    {{#unless user_has_email_set }}
+    <div class="tip">{{t "You must add an email address to your account to be able to invite users to Zulip." }}</div>
```

Comment:
I wonder if this could just use the same "to access this feature" language we use in the settings pages? Mostly just thinking about extra work for translators.